src/styles/blocks/card.scss

```diff
@@ -0,0 +1,123 @@
+.cards {
+  display: grid;
+  grid-gap: 46px 48px;
+  padding: 50px $space-40;
+}
+
+.card {
+  margin-bottom: 0;
+  width: 200px;
+
+  background-color: $button-light-color;
+  border-radius: 5px;
+  border: 1px solid $border-color;
+}
+
+.card > img {
```

Comment:
it is better to use class for styles. Add class for image and use it

Reply:
Hi, thanks for your feedback. That was my original idea, but unfortunately it's not possible to have more than 2 classes in 1 element.